pom.xml

```diff
@@ -6034,7 +6034,8 @@
                     <!-- Exclusions are tolerated for nightly "date-based" releases only -->
                     <excludes>
                       <exclude>org.nuxeo:nuxeo-ftest:*:zip</exclude>
-                      <!-- <exclude>org.nuxeo.connect:nuxeo-connect-client</exclude> -->
+                      <!-- TODO NXBT-1765 Disable the next line after the release of Nuxeo Connect Client 1.5.1 -->
```

Comment:
This is not the right way to proceed, this TODO comment is useless and wrong.
NXP and NXBT must not be hardly coupled.
Here, you only have to create a blocker NXP ticket for 9.2 requiring an alignment on 1.5.1.
The release of connect client 1.5.1 will happen at that time, not before. Unless it has its own reasons, unrelated to the platform.

see https://jira.nuxeo.com/browse/NXBT-1765?focusedCommentId=301527